### PR TITLE
Spotify Provider to ASP.NET Core 2.0

### PR DESCRIPTION
**Pull request by 0xNF**
Migrate the Spotify provider to ASP.NET Core 2.0

### CODE REVIEW

**Review by TheUltimateC0der — APPROVED** [review]

**Comment by kevinchalet** [comment]
Thanks for your PR. I took a brief look and I see many inconsistencies with the other providers (e.g we don't want to keep the authentication helper if we can avoid it and use the claims actions API).

Please fix the inconsistencies (including styling differences) and squash your PR.

File: src/AspNet.Security.OAuth.Spotify/AspNet.Security.OAuth.Spotify.csproj [hunk 1]
```diff
@@ -2,19 +2,16 @@
 
   <Import Project="..\..\build\packages.props" />
 
-  <PropertyGroup>
```
Comment (kevinchalet) [inline src/AspNet.Security.OAuth.Spotify/AspNet.Security.OAuth.Spotify.csproj]: Remove the format change please.

File: src/AspNet.Security.OAuth.Spotify/AspNet.Security.OAuth.Spotify.csproj [hunk 2]
```diff
@@ -2,19 +2,16 @@
...
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+ <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup>
     <Description>ASP.NET Core security middleware enabling Spotify authentication.</Description>
-    <Authors>Adam Reisinger</Authors>
+    <Authors>Adam Reisinger; 0xNF</Authors>
```
Comment (kevinchalet) [inline src/AspNet.Security.OAuth.Spotify/AspNet.Security.OAuth.Spotify.csproj]: Remove the extra space.

File: src/AspNet.Security.OAuth.Spotify/SpotifyAuthenticationExtensions.cs [hunk 3]
```diff
@@ -7,64 +7,74 @@
...
 
-namespace Microsoft.AspNetCore.Builder
+namespace Microsoft.Extensions.DependencyInjection
 {
     /// <summary>
     /// Extension methods to add Spotify authentication capabilities to an HTTP application pipeline.
     /// </summary>
     public static class SpotifyAuthenticationExtensions
     {
+
```
Comment (kevinchalet) [inline src/AspNet.Security.OAuth.Spotify/SpotifyAuthenticationExtensions.cs]: Remove.

File: src/AspNet.Security.OAuth.Spotify/SpotifyAuthenticationOptions.cs [hunk 4]
```diff
@@ -14,17 +16,24 @@ namespace AspNet.Security.OAuth.Spotify
     /// </summary>
     public class SpotifyAuthenticationOptions : OAuthOptions
     {
-        public SpotifyAuthenticationOptions()
+        public SpotifyAuthenticationOptions()
```
Comment (kevinchalet) [inline src/AspNet.Security.OAuth.Spotify/SpotifyAuthenticationOptions.cs]: Remove the extra space, please.

**Review by kevinchalet — CHANGES_REQUESTED** [review]

**Comment by kevinchalet** [comment]
Minor a few remarks, it looks fine.

When you're done, please rebase and squash your PR into a single commit named `Migrate the Spotify provider to ASP.NET Core 2.0` :+1:

**Comment by 0xNF** [comment]
Sure - I'll get the changes done within the day.

**Comment by kevinchalet** [comment]
Merged, thanks for your contribution @0xNF! :clap: 

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/commit/356d626ad6713c49d81e1e92f0d16c68657ce955

**Pull request closed without merging**